pages/study-groups-test.html

```diff
@@ -273,372 +279,7 @@ <h2 class="section-heading">Study Groups</h2>
-            }else if (person.technology === 'JavaScript'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '12 am') {
-              jsPart12am.push(person.name);
-            }else if (person.technology === 'JavaScript'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '6 am') {
-              jsPart6am.push(person.name);
-            }else if (person.technology === 'JavaScript'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '12 pm') {
-              jsPart12pm.push(person.name);
-            }else if (person.technology === 'JavaScript'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '6 pm') {
-              jsFull6pm.push(person.name);
-            }
-            if (person.technology === 'React'
-                && person.commitment === 'full time - 6 hours a day'
-                && person.beginning === '12 am') {
-              reactFull12am.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'full time - 6 hours a day'
-                      && person.beginning === '6 am') {
-              reactFull6am.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'full time - 6 hours a day'
-                      && person.beginning === '12 pm') {
-              reactFull12pm.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'full time - 6 hours a day'
-                      && person.beginning === '6 pm') {
-              reactFull6pm.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '12 am') {
-              reactPart12am.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '6 am') {
-              reactPart6am.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '12 pm') {
-              reactPart12pm.push(person.name);
-            }else if (person.technology === 'React'
-                      && person.commitment === 'part time - 3 hours a day'
-                      && person.beginning === '6 pm') {
-              reactFull6pm.push(person.name);
-            }
-          });
-        }
-    
-      </script>
+    <!-- 6/9/2019 Chris L: Created an external js file in the js folder, Script to sort study groups-->

```

Comment:
This comment probably belongs in the pull request rather than in the file or the commit messages to make it easier to review what was done.  The note that it's a script for sorting study groups is fine though so either way.